normandy/classifier/tests/test_api.py

```diff
@@ -56,11 +51,26 @@ def test_it_filters_by_locale(self, client):
         recipe_names = set(r['name'] for r in res.data['recipes'])
         assert recipe_names == {'german', 'any', 'both'}
 
+    def test_it_filters_by_locale_with_json(self, api_client):
```

Comment:
Why did you add this test? Is there something special about the json output of this api?